### PR TITLE
project complete and apply for checkup

### PR DESCRIPTION
**Pull request by ajkacca457**
We have completed the project and now we are submitting the project for review.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -28,11 +28,14 @@ nav {
...
 .iconu {
   color:red;
+  margin-right: 3%;
+  transform: scale(2);
 }
-.icon { transform: scale(1.3); }
+
 
 .notification {
   display: flex;
```
Comment (bertil291utn) [inline style.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `display:flex;` properties in entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display: flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### General requeriments

- [x] Next time please add at least 10 commit messages, bc it seems like you've just copied from another project if is not the case forgive me.

### Specific req.

- [x] It shows the recommended video's thumbnail preview ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/sidebar_thumbnail_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

image

- Using Inefficient CSS Selectors[1]

    When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

   This can be avoided by not using IDs within the selectors as they are directly specific.

  ```CSS
  .logo,.inputs,.notification {
    flex:1;
  }
  ```

  - [x]  Please keep `.notification` class and delete the left unnecessary rules

  <hr>

  _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_ 
  
### OPTIONAL

* [x] Encourage the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document
* [x] Encourage the use of the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer

### Summary

- Please take in mind the number of commits next time 
- Don't forget to work with inline comments below this comment box.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: style.css (hunk 1 above)
Comment (ajkacca457) [reply style.css]: commits
Hey .. Thank you very much for your review. We have learned a lot from your constructive way of pointing out our errors. This is just to inform you that we did commit messages plenty of time but the commit messages are not appearing in GitHub for some reason that we don't know. This is the first time we have worked with branches and probably we made mistakes while pushing branches to Github. We will check all the errors and after fixing it we will request for a recheck. Thank you very much for your quick feedback once again.
PS: I have attached all of our commits which are on display in our main branch. For some reason, these are not showing in the branch that we shared.

Comment (ajkacca457) [reply style.css]: Hei,
I shared our project link with my standup group and all of them can see the thumbnails picture for the recommended video just fine. I have also checked it by myself with another laptop and there was no that kind of issue that you mentioned.

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

### Hi team, you guys did a very good job implementing those previously requested changes.

#### Overall the project looks really nice and I can see that you guys are working hard on it, and don't worry, you're almost done, there are just some required changes that you guys have to implement to proceed with my approval:


- [x] Remove all the unnecessary repeated code (  [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/) )


Both of this code does exactly the same thing, yet they're at separated classes, kindly fix that.

image
&
image


`align-items: center;` appear 13 times, whenever possible, try to create a single class that deals with that styling.
image

**Please make the required changes and resubmit for a new code review once you are ready**

As always, I wish you all the very best and see you both at the top🚀

  

<hr>

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello @ajkacca457 and @Abhigyan001 :wave: 

## Congratulations, your project is approved, you may merge the changes. Good luck in your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**